Source/JavaScriptCore/runtime/JSGlobalObject.cpp

```diff
@@ -1796,9 +1797,11 @@ capitalName ## Constructor* lowerName ## Constructor = featureFlag ? capitalName
         m_objectPrototypeSymbolReplaceMissWatchpoint->install(vm);
     }
 
-    tryInstallArraySpeciesWatchpoint();
+    installArraySpeciesWatchpoint();
     catchScope.assertNoException();
 
+    installSaneChainWatchpoints();
```

Comment:
nit: is it worth to call `installSaneChainWatchpoints();` before `catchScope.assertNoException();`?